algosdk/future/transaction.py

```diff
@@ -460,6 +483,7 @@ def __init__(
         lease=None,
         rekey_to=None,
         nonpart=None,
+        state_proof_ID=None,
```

Comment:
nit: this SDK uses the msgpack name of the fields in most places, so `sprfkey` would be more appropriate